saw-remote-api/src/SAWServer/ProofScript.hs

```diff
@@ -154,7 +154,7 @@ makeSimpset params = do
         v <- getServerVal n
         case v of
           VSimpset ss' -> return (merge ss ss')
-          VTerm t -> return (addSimp (ttTerm t) ss)
+          VTerm t -> return (addSimp (ttTerm t) Nothing ss) -- TODO! making rewrite rules from terms!
```

Comment:
Maybe we should just remove this case. I don't think it'll break any existing proofs, and we can add convenience functions for proving and then making simpsets from terms in the Python client.